requirements.txt

```diff
@@ -6,3 +6,4 @@ cqc==3.2.2
 simulaqron==3.0.15
 eqsn==0.0.8
 qutip==4.5.2
+scipy~=1.5.4
```

Comment:
Let's use a hard requirement `scipy==1.5.4`

Reply:
got it